qa/src/test/resources/kapua-datastore-transport-client-setting.properties

```diff
@@ -0,0 +1,32 @@
+###############################################################################
+# Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     Eurotech - initial API and implementation
+#
+###############################################################################
+
+#
+# Elasticsearch Properties
+#NOTE: the datastore client provider implementation must provide:
+#      - a static init method (that initializes the client)
+#      - a static getInstance method (that returns the instance)
+datastore.elasticsearch.client.provider=org.eclipse.kapua.service.datastore.client.embedded.EsEmbeddedClientProvider
+#datastore.elasticsearch.client.provider=org.eclipse.kapua.service.datastore.client.transport.EsTransportClientProvider
+
+#timeouts
+datastore.query.timeout=60000
+datastore.scroll.timeout=60000
+
+datastore.elasticsearch.cluster=kapua-datastore
+datastore.elasticsearch.node=192.168.33.10
```

Comment:
It is not required by the standard build since we are using the embedded node but for a test against a real environment it should be specified (or replaced by the node list property).
So I prefer to keep it into the configuration file or at least leave it as comment so it will be easy to configure it without looking at the documentation and/or code to find the proper paramenter name.
Ok, your comment regarding the qa module so if someone wants to execute qa against an external ES he should has a proper code knowledge and he knows which properties should be set.
Do you agree if I leave the paramenter in the file as comment?